### PR TITLE
Fix Dictionary documentation by renaming Builtin.dll to DesignScriptBuiltin.dll

### PR DESCRIPTION
**Pull request by pboyer**
### Purpose

There was already a XML documentation file known as Builtin.xml that collided with Builtin.dll's docs.

Here we rename that dll to eliminate this and any future collisions.

capture

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@aparajit-pratap

### CODE REVIEW

**Comment by saintentropy** [comment]
LGTM

**Comment by saintentropy** [comment]
screen shot 2018-03-06 at 5 34 27 pm

@pboyer Is this the tooltip we want?

File: src/Libraries/DesignScriptBuiltin/DesignScriptBuiltin.csproj [hunk 1]
```diff
@@ -8,26 +8,28 @@
...
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
+    <OutputPath>..\..\..\bin\AnyCPU\Debug\</OutputPath>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DocumentationFile>..\..\..\bin\AnyCPU\Debug\en-US\DesignScriptBuiltin.xml</DocumentationFile>
```
Comment (mjkkirschner) [inline src/Libraries/DesignScriptBuiltin/DesignScriptBuiltin.csproj]: 👍

**Comment by aparajit-pratap** [comment]
@pboyer @mjkkirschner @saintentropy Please do not merge this PR. I have a solution where we needn't rename the assembly and have also got documentation working on builtin nodes. Will send out the PR soon.

**Comment by pboyer** [comment]
Closing for #8624

**Pull request closed without merging**